src/components/markdown/ProblemsList/DivisionList/contest_to_points.json

```diff
@@ -173,13 +173,16 @@
       0.9545454545454546, 0.9756944444444444, 0.9131944444444444
     ],
     "2022 December": [
-      0.8464285714285714, 0.776190476190477, 0.9650000000000002
+      0.8442028985507246, 0.7729468599033824, 0.9644927536231885
     ],
-    "2023 January": [0.8811594202898547, 0.9847826086956522, 1.0],
+    "2023 January": [0.8757575757575754, 0.984090909090909, 1.0],
     "2023 February": [
       0.9703947368421053, 0.9712918660287081, 0.951417004048583
     ],
-    "2023 US Open": [0.9057239057239058, 0.998217468805704, 0.8200757575757576]
+    "2023 US Open": [0.9057239057239058, 0.998217468805704, 0.8200757575757576],
+    "2023 December": [
+      0.9772727272727273, 0.9745454545454547, 0.9920454545454546
+    ]
```

Comment:
wait what's this whole file for

Reply:
<img width="900" alt="image" src="https://github.com/cpinitiative/usaco-guide/assets/65467530/31a2e7e5-5bbe-4367-9fca-b42ff6e460cd">
its like for the percent column